test/notificationSpec.js

```diff
@@ -44,6 +44,10 @@ describe('Notifications', function () {
     })
 
     afterEach(function * () {
+      if (nock.pendingMocks() && nock.pendingMocks().length) {
```

Comment:
Fixed in https://github.com/interledger/five-bells-connector/commit/4d75c2d1785199f9b3793314a2cba4a27ce47ef4